### ISSUE TITLE
Deploy Lesspod.org (to Firebase)

### ISSUE THREAD

**Issue by Rajan** [Rajan/lesspod#45]

**Comment by Rajan** [comment]
@s-kris I've deployed it to firebase and able to create an account. There seems to be an issue with login. Can you look? 

```
deployment target is firebase
Login.vue:107 email: ********  password: *********
Login.vue:115 deployment target is firebase
Login.vue:169 Error getting documents:  TypeError: Cannot read property 'data' of undefined
    at Login.vue:163
```

**Comment by s-kris** [comment]
It's working. I created a new account and tested the login.

**Comment by s-kris** [comment]
fixed: 4126b8bd83fded57ce5e2f7c6313fcd260f963f3
